dist/rpm/os-autoinst.spec

```diff
@@ -46,14 +50,24 @@ Source0:        %{name}-%{version}.tar.xz
 %if %{with spellcheck}
 # The following line is generated from dependencies.yaml
 %define spellcheck_requires aspell-en aspell-spell perl(Pod::Spell)
+%define spellcheck_make_args_for_autotools %{nil}
 %else
 %define spellcheck_requires %{nil}
+%define spellcheck_make_args_for_autotools CHECK_DOC=0
 %endif
 # The following line is generated from dependencies.yaml
-%define test_requires %build_requires %main_requires %spellcheck_requires perl(Benchmark) perl(Devel::Cover) perl(FindBin) perl(Pod::Coverage) perl(Test::Exception) perl(Test::Fatal) perl(Test::Mock::Time) perl(Test::MockModule) perl(Test::MockObject) perl(Test::Mojo) perl(Test::More) perl(Test::Output) perl(Test::Pod) perl(Test::Strict) perl(Test::Warnings) >= 0.029 perl(YAML::PP) procps python3-setuptools python3-yamllint qemu qemu-tools qemu-x86
+%define test_base_requires %main_requires perl(Benchmark) perl(Devel::Cover) perl(FindBin) perl(Pod::Coverage) perl(Test::Exception) perl(Test::Fatal) perl(Test::Mock::Time) perl(Test::MockModule) perl(Test::MockObject) perl(Test::Mojo) perl(Test::More) perl(Test::Output) perl(Test::Pod) perl(Test::Strict) perl(Test::Warnings) >= 0.029 procps python3-setuptools qemu qemu-tools qemu-x86
+# The following line is generated from dependencies.yaml
+%define test_legacy_requires %build_legacy_requires %test_base_requires
+# The following line is generated from dependencies.yaml
+%define test_requires %build_requires %spellcheck_requires %test_base_requires perl(YAML::PP) python3-yamllint
 # The following line is generated from dependencies.yaml
 %define devel_requires %test_requires perl(Devel::Cover) perl(Devel::Cover::Report::Codecov) perl(Perl::Tidy)
+%if 0%{?suse_version} == 1315
```

Comment:
where did you get this exact-match check from? I guess it should be `sle_version` according to https://en.opensuse.org/openSUSE:Build_Service_cross_distribution_howto

Reply:
I've got it from https://en.opensuse.org/openSUSE:Build_Service_cross_distribution_howto